### PR TITLE
INFINITY-3193: Revert "ELASTIC-159: Upgrade base tech to ES 6.1.3 (#2078)"

### PR DESCRIPTION
**Pull request by mpereira**
ESRI is currently is on Elastic 5.5.x. They can't move to 6.1.3 yet, probably because of the necessity to reindex data.

This made us decide to first release GA without ES6 so as to give them the latest security and rack awareness features from 1.11. We can then release Elastic 6.1.x right after the GA.

### CODE REVIEW

File: frameworks/elastic/src/main/dist/svc.yml [hunk 1]
```diff
@@ -49,17 +49,7 @@ pods:
...
-
-          export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jdk*/jre/)
-          ./elastic-scheduler/java-dns-ttl.sh
-          ./elastic-scheduler/install-plugins.sh
-          ./elastic-scheduler/setup-keystore-bootstrap-password.sh
-          ./elastic-scheduler/setup-java-policy.sh
-          ./bootstrap
-
-          exec ./elasticsearch-{{ELASTIC_VERSION}}/bin/elasticsearch
+        cmd: "export JAVA_HOME=$(ls -d $MESOS_SANDBOX/jdk*/jre/) && ./elastic-scheduler/java-dns-ttl.sh && ./elastic-scheduler/install-plugins.sh && ./elastic-scheduler/setup-java-policy.sh &&  ./bootstrap && exec ./elasticsearch-{{ELASTIC_VERSION}}/bin/elasticsearch"
```
Comment (elezar) [inline frameworks/elastic/src/main/dist/svc.yml]: Do we want to do a clean revert, or could we leave the "improvements" such as this?

File: frameworks/elastic/src/main/dist/install-plugins.sh [hunk 2]
```diff
@@ -20,7 +20,7 @@ if [ "$XPACK_ENABLED" = true ]; then
 fi
 
 if [ -n "$STATSD_UDP_HOST" ]; then
-    STATSD_PLUGIN="file://$MESOS_SANDBOX/elasticsearch-statsd-$ELASTIC_STATSD_VERSION.zip"
+    STATSD_PLUGIN="file://$MESOS_SANDBOX/elasticsearch-statsd-$ELASTIC_VERSION.0.zip"
```
Comment (elezar) [inline frameworks/elastic/src/main/dist/install-plugins.sh]: This is another thing that we could consider leaving in (or adding as an additional commit on top of the revert)?

File: frameworks/elastic/src/main/dist/svc.yml (hunk 1 above)
Comment (mpereira) [reply frameworks/elastic/src/main/dist/svc.yml]: @elezar I pushed a commit that makes this PR keep all non-actual-upgrade improvements: https://github.com/mesosphere/dcos-commons/pull/2274/commits/31a2cb45b47861c7c55d5013485378bc951b0551

File: frameworks/elastic/versions.sh [hunk 3]
```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env bash
 
-export TEMPLATE_ELASTIC_VERSION="6.1.3"
-export TEMPLATE_ELASTIC_STATSD_VERSION="${TEMPLATE_ELASTIC_VERSION}.0"
-export TEMPLATE_SUPPORT_DIAGNOSTICS_VERSION="6.3"
+export TEMPLATE_ELASTIC_VERSION="5.6.5"
+export TEMPLATE_SUPPORT_DIAGNOSTICS_VERSION="6.2"
```
Comment (elezar) [inline frameworks/elastic/versions.sh]: We still need the `TEMPLATE_ELASTIC_STATSD_VERSION` setting here.

Comment (mpereira) [reply frameworks/elastic/versions.sh]: https://github.com/mesosphere/dcos-commons/pull/2274/commits/d008a4c985cddc0d75834994fa9a0b7eed0560b2

**Pull request merged**